hadoop-hdds/framework/src/main/java/org/apache/hadoop/hdds/protocol/SCMSecurityProtocol.java

```diff
@@ -62,6 +63,18 @@ String getDataNodeCertificate(
   String getOMCertificate(OzoneManagerDetailsProto omDetails,
       String certSignReq) throws IOException;
 
+
+  /**
+   * Get SCM signed certificate for OM.
+   *
+   * @param scmNodeDetails       - DataNode Details.
+   * @param certSignReq     - Certificate signing request.
+   * @return String         - pem encoded SCM signed
+   *                          certificate.
+   */
+  String generateSCMPeerCertificate(ScmNodeDetailsProto scmNodeDetails,
```

Comment:
Yes, the other APIs should also start with generate, the correct naming is to start with generate. I will use generate for this new API. I can open a new Jira to change  for others.

Reply:
Can we name it `get...` now, and rename with the rest?  I know it's a bit more work, but there are some benefits:

1. They are consistent with one another both before and after the rename.
2. If renaming should turn out to be not possible (eg. because it might introduce incompatibility) or not desired (eg. someone might argue that `get` reflects the operation more closely), then we are not stuck with inconsistent names.

Reply:
I don't see why it will be an incompatible change. For now, I followed ur suggestion